### PR TITLE
7.8 release updates

### PR DESCRIPTION
**Pull request by benskelker**
DO NOT MERGE THIS PR UNTIL RELEASE DAY

Updates for the 7.8 release.

[Preview](https://docs_1868.docs-preview.app.elstc.co/guide/index.html)

### CODE REVIEW

**Comment by benskelker** [comment]
Hi @gtback @lcawl @jrodewig - can one of you review this. Thanks

File: conf.yaml [hunk 1]
```diff
@@ -878,7 +878,7 @@ contents:
             index:      app-search-docs/index.asciidoc
             private:    1
             current:    *stackcurrent
-            branches:   [ 7.7 ]
+            branches:   [ 7.8, 7.7 ]
             chunk:     1
             tags:       App Search/Guide
             subject:    App Search
```
Comment (benskelker) [inline conf.yaml:884]: @chriscressman - can you check this is correct for Enterprise Search? Thanks

**Comment by benskelker** [comment]
@elasticmachine, run elasticsearch-docs/doc rebuild

**Comment by benskelker** [comment]
@webmat should the ECS version be updated in this file https://github.com/elastic/docs/blob/master/shared/versions/stack/7.8.asciidoc

**Comment by benskelker** [comment]
@elasticmachine run elasticsearch-ci/docs rebuild

File: conf.yaml (hunk 1 above)
Comment (chriscressman) [reply conf.yaml:884]: Yes, looks good :rocket:

**Comment by webmat** [comment]
@benskelker Yes, should be 1.5, thanks :-)

**Comment by benskelker** [comment]
@elasticmachine run elasticsearch-ci/docs rebuild

**Comment by benskelker** [comment]
@elasticmachine run elasticsearch-ci/docs rebuild

**Comment by benskelker** [comment]
@elasticmachine run elasticsearch-ci/docs rebuild

**Pull request merged**